cli/internal/util/printf.go

```diff
@@ -21,11 +22,11 @@ func Sprintf(format string, args ...interface{}) string {
 }
 
 func Printf(format string, args ...interface{}) {
-	fmt.Fprint(os.Stderr, os.Expand(fmt.Sprintf(format, args...), replace))
+	Fprintf(os.Stderr, format, args...)
```

Comment:
Is there a reason for removing `os.Expand`?

Reply:
> replace `os.Expand` with `Sprintf()` which automatically formats and expands